CommonItem/src/main/java/com/github/sef24sp4/common/item/lootTable/LootTable.java

```diff
@@ -0,0 +1,93 @@
+package com.github.sef24sp4.common.item.lootTable;
+
+import com.github.sef24sp4.common.item.CommonItem;
+import com.github.sef24sp4.common.item.ItemRarity;
+import com.github.sef24sp4.common.item.ItemSPI;
+
+import java.util.*;
+
+public class LootTable {
+	private final Map<ItemRarity, Double> itemChances;
+	private final Collection<ItemSPI> itemProviders;
+
+	/**
+     * Constructs a new LootTable with the given item chances.
+     * The item providers are loaded using the ServiceLoader mechanism.
+     *
+     * @param itemChances a map from {@link ItemRarity} to their chances of type double.
```

Comment:
I think it should read something like?? 
> A map which maps {@link ItemRarity} to a `chance`. 

Please also specify valid values of for `chance`